glossary/sprint.md

```diff
@@ -0,0 +1,3 @@
+A sprint, in conventional terms, is "an act or short spell of running at full speed over a short distance". For New Atoms, a sprint, similarly, indicates a short duration of steady working that is fixed in advance (at this point, around two days).
+
+At the beginning of a sprint, the team votes on which [achievables](../glossary/achievable.md) will be put into "Next" on the [achievables board](https://github.com/newatoms/guides/tree/ready/board-guide). During a sprint, each team member works consistently on achievables (also called "cards") from the *Next* column of the achievables board and reviews the achievables fulfilled by other team members. At the end of a sprint, there is a demo during which we explain what we've done and why.
```

Comment:
"... on achievables (also called "cards") .. "
I would remove cards (bit confusing) and maybe emphasize that we only work on the voted in achievables ?

"demo" should actually have it's own glossary probably but that might not be a priority now :)